compiler/resolution/functionResolution.cpp

```diff
@@ -8651,10 +8651,38 @@ static Type* moveDetermineRhsTypeErrorIfInvalid(CallExpr* call) {
           if (rhsFn->hasFlag(FLAG_PROMOTION_WRAPPER))
             rhsName = unwrapFnName(rhsFn);
 
-          USR_FATAL(userCall(call),
+          // this deprecation-specific error message should be removed
```

Comment:
Let's put a `TODO: <date>` on the start of this comment so it stands out a bit more